packages/webapp/src/components/Navigation/TopMenu/TopMenu.jsx

```diff
@@ -131,18 +135,39 @@ const TopMenu = ({ history, isMobile, showNavigation, onClickBurger }) => {
   });
 
   const floaterMenu = (
-    <Menu
-      id="profile-menu"
-      anchorEl={profileIconRef.current}
+    <Popper
       open={openMenu}
-      onClose={closeMenu}
-      MenuListProps={{
-        'aria-labelledby': 'profile-navigation-button',
+      anchorEl={profileIconRef.current}
+      role={undefined}
+      placement="bottom-start"
+      transition
+      disablePortal
+      sx={{
+        top: '8px !important',
       }}
-      classes={{ list: styles.menuList, root: styles.menuRoot }}
     >
-      {menuItems}
-    </Menu>
+      {({ TransitionProps, placement }) => (
+        <Grow
+          {...TransitionProps}
+          style={{
+            transformOrigin: placement === 'bottom-start' ? 'left top' : 'left bottom',
+          }}
+        >
+          <Paper>
```

Comment:
I'm not sure if it's a good idea, but `margin-top: 8px` can be added here (or menuRoot?) to avoid `!important`.

Reply:
nice thank you! -- I was fixated on `top`